tests/e2e/ctl_v3_grpc_test.go

```diff
@@ -41,38 +41,38 @@ func TestAuthority(t *testing.T) {
 		{
 			name:                   "http://domain[:port]",
 			clientURLPattern:       "http://localhost:${MEMBER_PORT}",
-			expectAuthorityPattern: "localhost:${MEMBER_PORT}",
+			expectAuthorityPattern: "localhost",
```

Comment:
This is suspicious and possibly a breaking change. Please check https://github.com/etcd-io/etcd/issues/13192. Having just host is a correct authority format (as port is optional), however I think it can break some setups that depend on that. 
Before merging we should understand the grpc change that lead to that.

Reply:
The change https://github.com/grpc/grpc-go/pull/5748

Reply:
See the commit message (Pasted below in case anyone miss it) and https://github.com/grpc/grpc-go/pull/5748#issuecomment-1386381838

```
ServerName field of the resolver returned address takes precedence
over Host field of CallHdr to determine the :authority header.
Refer to https://github.com/grpc/grpc-go/pull/5748
```